src/okareo/reporter.py

```diff
@@ -0,0 +1,46 @@
+import json
+import os
+
+from okareo_api_client.models import TestRunItem
+
+
+class JSONReporter:
+    """
+    A simmple class to define a json reporter for output of results.
+    This class assumes that it is being used within the context of the CLI.
+    """
+
+    def __init__(self, eval_runs: list[TestRunItem]):
+        self.eval_runs = eval_runs
+
+    def log(self) -> None:
+        """
+        Logs the results of the evaluation runs into a file based on OKAREO_REPORT_DIR.
+        """
+        okareo_report_dir = os.environ["OKAREO_REPORT_DIR"]
```

Comment:
if this env var isn't defined, does this just write the report to the current directory?

Reply:
This env_var is from the CLI.  When it is not present, this outputs to the console.